code/style.css

```diff
@@ -1,4 +1,229 @@
-html {
-  background: #0025ff;
+* {box-sizing: border-box;}
+
+/* just to make the whole site in center*/
+.wrapper {
+ max-width: 960px;
+ margin: auto;
+ font-family: Helvetica, sans-serif;
+}
+
+/*header styling*/
+.header{
+  margin: 0;
+  padding: 0;
+}
+/* header top row styling*/
+.header-row1{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+}
+.menu-bars,.menu-search {
+  background-color:#FFCAD4;
+  height:fit-content;
+  border: none;
+  color: white;
+  font-size: 20px;
+  cursor: pointer;
+} 
+.logo{ 
+display: none;
+}
+.site-name {
+  font-family: 'Finger Paint', cursive;
+  font-size: 1.5rem;
+  color:#cdeac0;
+  padding: 0; 
+}
+.site-name span{
+  color: #2c6e49;
+}
+/* header menue row styling*/
+.header-row2{
+  display: none;
+}
+
+/* site content styling*/
+.content{
+  background-color: #d8e2dc;
+ }
+.row{
+  display: flex;
+  flex-direction: column;
+  align-items:stretch;
+}
+.top-fig{
+  display: block;
+  position:relative 
+} 
+.top-fig p{
+  display: none;
+  color:#fff;
+  background-color:#c9c9c9;
+  width: 100%;
+  padding: 10px;
+}
+.top-fig:hover p{
+  position:absolute; top:75%; left:0;
+  display: block;
+}
+.top-fig figcaption { 
+  position:absolute; Top:0; left:0; 
+  padding: 10px; 
+ }
```

Comment:
It is easy to read when you write all of the styling for one specific "thing" in your website. It gives me a sence of overview